app/presenters/iiif_print/iiif_manifest_presenter_behavior.rb

```diff
@@ -8,7 +8,9 @@ module IiifManifestPresenterBehavior
     attr_accessor :base_url
 
     def manifest_metadata
-      @manifest_metadata ||= IiifPrint.manifest_metadata_from(work: model, presenter: self)
+      # ensure we are using a SolrDocument
+      solr_doc = model.is_a?(SolrDocument) ? model : model.solr_document
+      @manifest_metadata ||= IiifPrint.manifest_metadata_from(work: solr_doc, presenter: self)
```

Comment:
With memoization, we likely want the following:

```ruby
@manifest_metadata ||= begin
	                 solr_doc = model.is_a?(SolrDocument) ? model : model.solr_document
                         IiifPrint.manifest_metadata_from(work: solr_doc, presenter: self)
                       end
```

In this way, we're not rebuilding the sol_doc.